modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -317,6 +317,8 @@ type TLSConfig struct {
 
 	// The server name (SNI) to use in TLS handshakes.
 	ServerName string `json:"server_name,omitempty"`
+
+	Renegotiation string `json:"renegotiation,omitempty"`
```

Comment:
Put a comment above this line to document it. Otherwise this field will have no documentation on the Caddy website. Especially important since it accepts 3 specific values.